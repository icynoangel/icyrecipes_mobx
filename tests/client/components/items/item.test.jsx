import React from 'react';
import ReactDOM from 'react-dom';
import Item from './../../../../src/client/js/components/items/item';
import {shallow} from 'enzyme';
import enzymeToJson from 'enzyme-to-json';

describe('<Item />', function() {
  this.item = {
    itemImage: '/boat.jpg',
    itemTitle: 'Boat',
    itemDescription: 'Lonely boat'
  };

  this.getComponent = () => {
    return <Item item={this.item} />;
  };

  it('Should render <Item />', () => {
    const wrapper = shallow(this.getComponent());
    expect(enzymeToJson(wrapper)).toMatchSnapshot();
  });
});
