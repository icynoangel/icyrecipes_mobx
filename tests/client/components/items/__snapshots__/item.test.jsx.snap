// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Item /> Should render <Item /> 1`] = `
<div
  className="item"
>
  <div
    className="item__image"
  >
    <img />
  </div>
  <div
    className="item__details"
  >
    <div
      className="item__title"
    />
    <div
      className="item__description"
    />
  </div>
</div>
`;
